Directory.Build.targets

```diff
@@ -61,20 +62,19 @@ We need to run a unity build to restore the test packages and for that reason we
 For that reason, Exec tasks to unity all ignore error. That means that subsequent tasks must exist to validate the command (i.e: parse logs, etc).
 Related: https://forum.unity.com/threads/6572-debugger-agent-unable-to-listen-on-27.500387/  -->
 
-  <!-- Build a player with IL2CPP: dotnet msbuild /t:UnityBuildPlayerIL2CPP -->
-  <Target Name="UnityBuildPlayerIL2CPP"
+  <!-- Build a player with IL2CPP: dotnet msbuild /t:UnityBuildStandalonePlayerIL2CPP -->
+  <Target Name="UnityBuildStandalonePlayerIL2CPP"
           Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'">
     <Error Condition="$(UnityRoot) == ''" Text="Couldn't find Unity."></Error>
 
     <Message Importance="High" Text="Building Player with IL2CPP."></Message>
 
     <Exec Command="&quot;$(UnityExec)&quot; -quit -batchmode -nographics -logFile - -projectPath $(UnitySampleProjectPath) -executeMethod $(StandaloneBuildMethod) -buildPath $(StandaloneBuildPath)"
-          IgnoreStandardErrorWarningFormat="true"
-          IgnoreExitCode="true"></Exec>
+          IgnoreStandardErrorWarningFormat="true"></Exec>
```

Comment:
Unity is returning `0` on successful builds on all my tests so I removed the `IgnoreExitCode="true"`.

So builds should break already based on Unity's return code now.